src/main/resources/presets/blocks/01-align.yaml

```diff
@@ -450,7 +450,7 @@ align-single-cell:
           type: kaligner2
           mapperNValue: 9
           mapperKValue: 1
-          floatingLeftBound: false
```

Comment:
Wow, this will affect many presets...
@PoslavskySV @mizraelson You have check this.